src/test/java/com/exasol/adapter/installer/Db2InstallerIT.java

```diff
@@ -0,0 +1,80 @@
+package com.exasol.adapter.installer;
+
+import static com.exasol.adapter.installer.VirtualSchemaInstallerConstants.*;
+
+import java.io.IOException;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.concurrent.TimeoutException;
+
+import org.apache.commons.cli.ParseException;
+import org.junit.jupiter.api.*;
+import org.testcontainers.containers.Db2Container;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+import com.exasol.adapter.installer.dialect.Dialect;
+import com.exasol.bucketfs.BucketAccessException;
+
+@Tag("integration")
+@Testcontainers
+class Db2InstallerIT extends AbstractIntegrationTest {
+    private static final String DB2_SCHEMA = "DB2INST1";
+    private static final int DB2_PORT = 50000;
+    public static final String DOCKER_IMAGE_REFERENCE = "ibmcom/db2:11.5.5.0";
+
+    @Container
+    private static final Db2Container DB2 = new Db2Container(DOCKER_IMAGE_REFERENCE);
+
+    @BeforeAll
+    static void beforeAll() throws SQLException {
+        final Statement sourceStatement = createStatement();
+        createSimpleTestTable(sourceStatement, DB2_SCHEMA);
+    }
+
+    private static Statement createStatement() throws SQLException {
+        return DB2.createConnection("").createStatement();
+    }
+
+    @Test
+    void testInstallVirtualSchema()
+            throws SQLException, BucketAccessException, TimeoutException, ParseException, IOException {
+        final String virtualSchemaName = "DB2_VIRTUAL_SCHEMA_1";
+        final String[] args = new String[] { //
+                "--" + JDBC_DRIVER_NAME_KEY, "jcc.jar", //
```

Comment:
Couldn't the installer download the driver?

Reply:
From a users perspective that would be very convenient.

Reply:
(I know it's not part of this PR. But still. If possible I think we should create an issue for it)

Reply:
Not for now, a lot of drivers require registration or accepting some licenses to download them. It can be a legal issue, only a very few drivers can be downloaded freely